fractions/microprofile/microprofile-restclient/src/main/java/org/wildfly/swarm/microprofile/restclient/RestClientFraction.java

```diff
@@ -15,17 +15,16 @@
  */
 package org.wildfly.swarm.microprofile.restclient;
 
+import static org.wildfly.swarm.spi.api.annotations.DeploymentModule.MetaInfDisposition.IMPORT;
+
 import org.wildfly.swarm.spi.api.Fraction;
 import org.wildfly.swarm.spi.api.Module;
 import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
 
-import static org.wildfly.swarm.spi.api.annotations.DeploymentModule.MetaInfDisposition.IMPORT;
-
 /**
  * @author Heiko Braun
  */
-@DeploymentModule(name = "org.wildfly.swarm.microprofile.restclient", slot = "deployment", metaInf = IMPORT,
-        export = true, services = Module.ServiceHandling.IMPORT)
+@DeploymentModule(name = "io.smallrye.restclient", metaInf = IMPORT, export = true, services = Module.ServiceHandling.IMPORT)
```

Comment:
Is this still needed or having the pom dependency with the module.xml is enough ?

Reply:
Based on Michal's feedback to my comment to hos PR I guess my comment can simply be ignored :-)